resources/views/user/workshop_tags.blade.php

```diff
@@ -2,7 +2,11 @@
 @if($user->hasEducationalInformation())
 @foreach ($user->workshops as $workshop)
     <span class="new badge {{ $workshop->color() }} scale-transition" id="user-workshop-{{ $user->id }}-{{ $workshop->id }}" data-badge-caption="">
-        <nobr>{{$workshop->name}} <span style="cursor: pointer;" onclick="deleteWorkshop({{ $user->id }}, {{ $workshop->id }})">&cross;</span></nobr>
+        <nobr>{{$workshop->name}} 
```

Comment:
BTW, would removing nobr look good/bad? Would it solve the too long name issue with Mendöl?

Reply:
I think multilined tags look bad. However, we can try switching from tables to divs.

Reply:
Before switching, let's think about different options.

Reply:
Maybe trimming long names would be enough.